<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoundingBoxTestDialog</class>
 <widget class="QDialog" name="BoundingBoxTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="_acceptButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_rejectButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QVTKWidget" name="_qvtkWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>400</width>
     <height>530</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollBar" name="_minXScrollBar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>-1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="_maxXScrollBar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="_minYScrollBar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>-1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="_maxYScrollBar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="_minZScrollBar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>-1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="_maxZScrollBar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="_minXLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Min X：</string>
   </property>
  </widget>
  <widget class="QLabel" name="_maxXLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Max X：</string>
   </property>
  </widget>
  <widget class="QLabel" name="_minYLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Min Y：</string>
   </property>
  </widget>
  <widget class="QLabel" name="_maxYLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Max Y：</string>
   </property>
  </widget>
  <widget class="QLabel" name="_minZLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Min Z：</string>
   </property>
  </widget>
  <widget class="QLabel" name="_maxZLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Max Z：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_minXLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_maxXLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_minYLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_maxYLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_minZLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_maxZLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
